DigitalLearningSolutions.Web/ViewModels/MyAccount/EditDetailsFormData.cs

```diff
@@ -90,5 +105,70 @@ protected EditDetailsFormData(EditDetailsFormData formData)
         public string? Answer5 { get; set; }
 
         public string? Answer6 { get; set; }
+
+        public string? ProfessionalRegistrationNumber { get; set; }
+
+        public YesNoSelectionEnum ProfessionalRegNumberSelectionAnswer { get; set; }

```

Comment:
I'm not a big fan of the abbreviation to Reg here, but agree it would be a very long property name without it. Would something like `HasProfessionalRegistrationNumber` work instead?

Reply:
Sorry, I have renamed it as suggested